src/controllers/portfolio/portfolio.ts

```diff
@@ -105,6 +106,10 @@ export class PortfolioController extends EventEmitter {
   // Holds the initial load promise, so that one can wait until it completes
   #initialLoadPromise: Promise<void>
 
+  #shouldShowConfettiModal: boolean = false
```

Comment:
Too generic here as well, and no context (in a comment) what this confetti modal is for.